compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/AddContinuationLowering.kt

```diff
@@ -641,17 +641,18 @@ private class AddContinuationLowering(private val context: JvmBackendContext) :
     private fun markSuspendLambdas(irElement: IrElement): Pair<List<SuspendLambdaInfo>, List<IrFunction>> {
         val suspendLambdas = arrayListOf<SuspendLambdaInfo>()
         val capturesCrossinline = mutableSetOf<IrCallableReference>()
-        val visitor = object : IrInlineReferenceLocator(context) {
+        val inlineReferences = mutableSetOf<IrCallableReference>()
+        irElement.acceptChildrenVoid(object : IrInlineReferenceLocator(context) {
             override fun visitElement(element: IrElement) {
                 element.acceptChildrenVoid(this)
             }
 
-            override fun handleInlineFunctionCallableReferenceParam(valueArgument: IrCallableReference) {
-                super.handleInlineFunctionCallableReferenceParam(valueArgument)
-                val getValue = (valueArgument as? IrGetValue) ?: return
+            override fun visitInlineReference(argument: IrCallableReference) {
```

Comment:
@pyos @ilmirus Looks like that this code now is called more times then before. is it make any sense (should it affect any test?)

Reply:
Oh, looks like that after 'inlineCallableReferenceToLambdaPhase' there are no such cases